project/Build.scala

```diff
@@ -816,6 +816,18 @@ object Build {
 
       previousArtifactSetting,
       mimaBinaryIssueFilters ++= BinaryIncompatibilities.Linker,
+      mimaBinaryIssueFilters ++= {
+        // Always exclude packages where we give no compatibility guarantee.
+        import com.typesafe.tools.mima.core.Problem
+        import com.typesafe.tools.mima.core.ProblemFilters.exclude
+
```

Comment:
There should either be no blank line here, or the whole `{}` block should have blank lines around it.

Otherwise LGTM.